### ISSUE TITLE
Error executing vim.schedule lua callback: .../pack/packer/start/nvim-tree.lua/lua/nvim-tree/view.lua:185: Vim(buffer):E86: Buffer 2 does not exist

### ISSUE THREAD

**Issue by kdheepak** [kyazdani42/nvim-tree.lua#572]
When start neovim without any buffers, nvim tree auto opens. Then when I load a session, and then move around between the buffers (using `:bnext`), I get the following error:

```
Error executing vim.schedule lua callback: .../pack/packer/start/nvim-tree.lua/lua/nvim-tree/view.lua:185: Vim(buffer):E86: Buffer 2 does not exist
```

I'm not sure exactly what is going on but I'm able to consistently reproduce it in one of my repositories.

**Comment by kyazdani42** [comment]
well i'm not sure how sessions work in neovim, i should maybe take a look to understand why there are so many issues with them

**Comment by B3RR10** [comment]
Unfortunately I have the same problem...

**Comment by vpeopleonatank** [comment]
I 'm facing the same issue

**Comment by kyazdani42** [comment]
should be fixed in `1.1.5` (4cc2abe3e238d282d4b7e25860033356a64e5a8e)
